### PR TITLE
feat: cfd-453 operator details

### PR DESCRIPTION
**Pull request by matt-twomey**
# Description

-   Allows operators to input their details in global settings

# Testing instructions

-   Test that validations are applied as expected (all fields mandatory, and contactNumber, email, url and postcode validations as per https://infinityworks-dft.atlassian.net/browse/CFD-435

# Type of change

-   [x] feat - A new feature
-   [ ] fix - A bug fix
-   [ ] docs - Documentation only changes
-   [ ] codestyle - Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
-   [ ] refactor - A code change that neither fixes a bug nor adds a feature
-   [ ] perf - A code change that improves performance
-   [ ] test - Adding missing tests or correcting existing tests
-   [ ] build - Changes that affect the build system or external dependencies (example scopes: gulp, broccoli, npm)
-   [ ] ci - Changes to our CI configuration files and scripts (example scopes: Travis, Circle, BrowserStack, SauceLabs)
-   [ ] chore - Other changes that don't modify src or test files
-   [ ] revert - Reverts a previous commit
-   [ ] content - Changes to content or copy

### CODE REVIEW

**Comment by JwanKhalaf** [comment]
In the `Navigation` file, you need to get it to show active state when on `Operator details` page.

image

File: shared/matchingJsonTypes.ts [hunk 1]
```diff
@@ -297,3 +297,14 @@ export interface Stop {
...
+
+export interface OperatorDetails {
+    operatorName: string;
+    contactNumber: string;
+    email: string;
+    url: string;
+    street: string;
+    town: string;
+    county: string;
+    postcode: string;
```
[the remark this answers is not in the document]
Comment (matt-twomey) [reply shared/matchingJsonTypes.ts:309]: Ah, so I believe postcode is one word
image

[the remark this answers is not in the document]
Comment (JwanKhalaf) [reply shared/matchingJsonTypes.ts:309]: You're right! Nice :+1: :)

**Review by Aaron-Collis-IW — APPROVED** [review]
Looks good to me, all the validation seems to be working

**Review by JwanKhalaf — APPROVED** [review]
Looks good :+1:

**Pull request merged**